src/libs/auth/RedirectFromOAuth.ts

```diff
@@ -0,0 +1,22 @@
+import { UserManager } from 'oidc-client-ts';
+import { Spinner } from '../../components/Spinner';
+import { OidcBroker } from './oidcBroker';
+
+const RedirectFromOAuth = (): JSX.Element => {
+  const userManager: UserManager = new UserManager(
+    OidcBroker.getUserManagerSettings()
+  );
+  const url = window.location.href;
+  const isSilent = window.location.pathname.startsWith(
+    '/redirect-from-oauth-silent'
+  );
+
+  if (isSilent) {
+    userManager.signinSilentCallback(url);
+  } else {
+    userManager.signinPopupCallback(url);
+  }
+  return Spinner;
```

Comment:
```suggestion
  return <Spinner />;
```

Reply:
Oh. I assumed it was based on the name. But if it was an element instead, that change will likely break other places it was used.